internal/printer/printer_test.go

```diff
@@ -407,6 +408,54 @@ import Widget2 from '../components/Widget2.astro';`},
   </head><body></body></html>`,
 			},
 		},
+		{
+			name: "script hoist with frontmatter",
+			source: `---
+---
+<script type="module" hoist>console.log("Hello");</script>`,
+			want: want{
+				imports:     "",
+				frontmatter: []string{},
+				styles:      []string{},
+				scripts:     []string{fmt.Sprintf(`{props:{"type":"module","hoist":true},children:%sconsole.log("Hello");%s}`, BACKTICK, BACKTICK)},
```

Comment:
As mentioned, we're retaining `type: "module"` and any other attributes. The runtime can remove `hoist` for us.